src/plugin.js

```diff
@@ -1,42 +1,72 @@
 import { createStore } from './store'
 
-// Vanilla Installer
+export const injectedPropertyName = 'stator'
 
-function lazyInject (Vue, key, setter) {
-  // Check if plugin not already installed
-  const installKey = `__stator_${key}_installed`
-  if (Vue[installKey]) {
-    return
-  }
+export function injectStator (Vue, options) {
+  lazyInject(Vue, injectedPropertyName, (vm) => {
+    return createStore(options, vm.$root)
+  })
 
-  Vue[installKey] = true
+  injectProperty(Vue, injectedPropertyName, '$state')
+  injectProperty(Vue, injectedPropertyName, '$getters')
+  injectProperty(Vue, injectedPropertyName, '$actions')
+}
 
-  // Call Vue.use() to install the plugin into vm
+export function lazyInject (Vue, key, setter) {
   Vue.use(() => {
-    if (!Vue.prototype.hasOwnProperty(key)) {
-      Object.defineProperty(Vue.prototype, key, {
-        configurable: true,
-        get () {
+    const propertyName = `$${key}`
```

Comment:
This changes the inserted property name from `_stator` to `$stator`. The reason for this is that I added some helper methods on the store object, so it becomes a viable option for the user to use.

Eg `this.$stator.registerModule()`